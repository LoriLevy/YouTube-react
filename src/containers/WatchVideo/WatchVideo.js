import React from 'react';
import './WatchVideo.scss';

export class WatchVideo extends React.Component {
  render() {
    return (
      <div>Watch Video component goes here</div>
    );
  }
}