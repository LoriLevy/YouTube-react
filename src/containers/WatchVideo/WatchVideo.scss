.eighty-percent {
    max-width: 80%;
}