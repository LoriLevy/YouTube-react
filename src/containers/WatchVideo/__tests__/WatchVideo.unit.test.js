import React from 'react';
import {shallow} from 'enzyme';
import {WatchVideo} from '../WatchVideo';

describe("WatchVideo", () => {
    test("it renders", () => {
      const wrapper = shallow(<WatchVideo />);
      expect(wrapper).toMatchSnapshot();
    });
  });