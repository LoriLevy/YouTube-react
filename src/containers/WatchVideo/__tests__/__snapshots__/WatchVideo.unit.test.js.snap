// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WatchVideo it renders 1`] = `
<div>
  Watch Video component goes here
</div>
`;
